src/Raven.Studio/typescript/models/database/tasks/ongoingTaskOlapEtlTransformationModel.ts

```diff
@@ -47,13 +49,13 @@ class ongoingTaskOlapEtlTransformationModel {
        ], false, jsonUtil.newLineNormalizingHashFunction);
     }
    
-    static empty(): ongoingTaskOlapEtlTransformationModel {
+    static empty(suggestedName: string): ongoingTaskOlapEtlTransformationModel {
```

Comment:
it's simply name  - not suggested name. Suggested name is an example of usage.

Reply:
additionally we might made this parameter optional

Reply:
done